client/components/data/query-preferences/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingPreferences } from 'state/preferences/selectors';
+import { fetchPreferences } from 'state/preferences/actions';
+
+class QueryPreferences extends Component {
+
+	componentWillMount() {
+		if ( ! this.props.isFetchingPreferences ) {
+			this.props.fetchPreferences();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+QueryPreferences.propTypes = {
+	requestingSites: PropTypes.bool,
```

Comment:
Assume this should be `isFetchingPreferences`, in which case a few notes:

- We should be consistent with how we apply "fetch" and "request" verbs, at least within the same state. The action types introduced in this branch are suffixed with "request" but the selectors and props use "fetch". I've leaned more toward "request" lately, since "fetch" restricts us more to only applying well for `GET` network requests.
- I've recommended omitting the `is` prefix for props, for consistency and to avoid naming conflicts. Consistency in that DOM properties are named without the prefix (e.g. `disabled` and `readOnly` vs. `isDisabled` and `isReadOnly`). Naming conflicts in that if a developer were to access the prop by `const { isFetchingPreferences } = this.props;`, they'd introduce a lint warning on [`no-shadow`](http://eslint.org/docs/rules/no-shadow) (conflicting with named dependency).